### DISCUSSION TITLE
Want resource URIs data in token api in Keycloak using User Access Token

### DISCUSSION THREAD

**Discussion by Rutvij07** [keycloak/keycloak#30688] [Q&A]
### Description

My curl Request 

`curl --location ‘{domain}/realms/{realm}/protocol/openid-connect/token’
–header ‘Content-Type: application/x-www-form-urlencoded’
–header 'Authorization: Bearer {token}
–data-urlencode ‘grant_type=urn:ietf:params:oauth:grant-type:uma-ticket’
–data-urlencode ‘audience={client_id}’
–data-urlencode ‘response_mode=permissions’ \`

Reponse 

`[
    {
        "scopes": [
            "read"
        ],
        "rsid": "f4cdcc0d-aebc-4b9f-8e4d-c89f4d48a5ac",
        "rsname": "read-terminal"
    }
]`

expected response 

`[
    {
        "scopes": [
            "read"
        ],
        "rsid": "f4cdcc0d-aebc-4b9f-8e4d-c89f4d48a5ac",
        "rsname": "read-terminal",
         "uris":['/Reports/list','/Reports/recurringReports']
    }
]`

**Comment by owaismohammed** [comment]
Did You find the answer @Rutvij07

**Comment by laopaoer-wallet** [comment]
Did You find the answer @Rutvij07 @owaismohammed

**Comment by laopaoer-wallet** [comment]
Dynamic Permission Control So need uris
